sdk/python/kfp/v2/dsl/importer_node_test.py

```diff
@@ -105,17 +105,17 @@ def test_build_importer_spec_with_invalid_inputs_should_fail(self):
           input_type_schema='title: kfp.Artifact',
           pipeline_param_name='param1',
           constant_value='some_uri')
-      self.assertEqual(
-          'importer spec should be built using either pipeline_param_name or'
-          'constant_value.',
-          str(cm))
+    self.assertEqual(
```

Comment:
Sorry for missing this in the previous PR. I think you can do
```
with self.assertRaisesRegexp(AssertionError, 'importer spec should ... '):
  ...
```
to make the code simpler

Reply:
Thanks!